examples/nextjs-with-typescript-v4-migration/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "nextjs-with-typescript-with-mui-styles",
+  "version": "5.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "post-update": "echo \"codesandbox preview only, need an update\" && yarn upgrade --latest"
+  },
+  "dependencies": {
+    "@emotion/cache": "^11.7.1",
+    "@emotion/react": "^11.9.0",
+    "@emotion/server": "^11.4.0",
+    "@emotion/styled": "^11.8.1",
+    "@mui/icons-material": "^5.0.0",
+    "@mui/material": "^5.0.0",
+    "@mui/styles": "^5.0.0",
```

Comment:
I think we should mark them as `latest`?

Reply:
@samuelsycamore I wonder if we need to mention in the migration guide that React 18 upgrade should be done after MUI upgrade?

cc @mnajdova does it make sense?

Reply:
> React 18 upgrade should be done after MUI upgrade

If you mean: React 18 upgrade should be done after `@mui/styles` is replaced with one of the alternatives, than yes, good point 👍